File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        String str;
        System.out.println(phone_number.length());
        StringBuffer sb = new StringBuffer(phone_number);
        for(int i=0; i<phone_number.length()-4;i++){
            sb.replace(i,i+1,"*");
        }
        
        answer = sb.toString();
        // for(int i=0;i<phone_number.length-4;i++){
        //     phone_number[i]="*";
        // }
        
        return answer;
    }
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.js
function solution(phone_number) {
    let answer = '';
    
    for(let i = 0; i < phone_number.length; i++){
        if(i + 4 >= phone_number.length){
            answer = answer + phone_number[i];
        }else{
            answer = answer + '*';
        }
    }
    return answer;
}